packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -343,6 +349,18 @@ export function shouldSetTextContent(type: string, props: Props): boolean {
 }
 
 export function getCurrentEventPriority(): * {
+  const currentEventPriority = fabricGetCurrentEventPriority();
+
+  if (currentEventPriority !== undefined) {
```

Comment:
So `null` would be the Default priority? That's how it ends up working in DOM, but just making sure that's intended.